Demo Documentation (#11)
dove-technology/commercetools-campaigns-connector-demo: README.md

	"code": "CODE"
}
```

When a call is made to update the cart, commercetools calls the service in the Dovetech connector (using an API extension) to validate the coupon code.

Valid coupon codes are added to the `dovetech-discounts-couponCodes` field.

If a coupon code isn't valid an error with status 400 is returned and the coupon code is not added to the cart.

Coupon codes can be removed from the cart by updating the `dovetech-discounts-couponCodes` custom field.

### Customers

You can simulate setting a customer on a cart by using the `/set-customer` page. Enter the email address of a customer in commercetools. This will then set the customer ID and email on the cart. You can then test out customer specific discounts (e.g. customer specific promo codes or loyalty schemes).

### Checkout

This demo uses the "Complete Checkout" mode for the commercetools checkout. See [Applications](https://docs.commercetools.com/checkout/payment-connectors-applications#applications) for more details.

You need a checkout app installed in your commercetools project to use this. See [Configure Checkout](https://docs.commercetools.com/checkout/configuring-checkout) for more details.

> The "Allow buyers to add discount codes" setting in the checkout app should be left disabled because this will use the commercetools discount codes functionality.

Reviewer: ```suggestion
The "Allow buyers to add discount codes" setting in the checkout app should be left disabled otherwise the commercetools discount codes functionality would be used instead of Dovetech's.
```
